go-controller/pkg/ovn/master.go

```diff
@@ -1022,3 +989,67 @@ func (oc *DefaultNetworkController) deleteHoNodeEvent(node *kapi.Node) error {
 	}
 	return nil
 }
+
+// addIPToHostNetworkNamespaceAddrsSet adds management port IP and node's
+// gateway-router-lrp-ifaddr to address_set created for HostNetworkNamespace.
+// This function gets called from both AddResource & UpdateResource to add IPs
+// to address_set for both local and remote zone nodes.
+func (oc *DefaultNetworkController) addIPToHostNetworkNamespaceAddrsSet(node *kapi.Node) error {
```

Comment:
nit: 
```suggestion
func (oc *DefaultNetworkController) addIPToHostNetworkNamespaceAddrSet(node *kapi.Node) error {
```
loose the s for better readability same with the delete func